js/exmo.js

```diff
@@ -1024,12 +1024,9 @@ module.exports = class exmo extends Exchange {
     }
 
     handleErrors (httpCode, reason, url, method, headers, body, response) {
-        if (typeof body !== 'string')
-            return; // fallback to default error handler
-        if (body.length < 2)
+        if (typeof response === 'undefined')
```

Comment:
I suggest that we use a slightly shorter version without string-comparisons:

```JavaScript
if (response === undefined) {
    // ...
}
```

Can you plz add this edit everywhere below?

Reply:
> If you think the former is better I can blast the latter with a regex.
That would be great! ) If you will, please do it as a standalone PR though, just to keep things separate. Thx! )